lib/sensu-plugins-nrpe/version.rb

```diff
@@ -1,8 +1,8 @@
-module SensuPluginsNRPE
+module SensuPluginsNrpe
   module Version
     MAJOR = 0
     MINOR = 0
-    PATCH = 1
+    PATCH = 2
```

Comment:
Leave the version bumps to the maintainers as there are still often other commits that have not been released and might be more than a patch or minor.